tests/ut_seash_savestate_nicermessage.py

```diff
@@ -0,0 +1,38 @@
+"""
+This unit test is to test error messages at these situations. One is using 
+savestate before having loaded at least a pubkey, and work as that identity.
+Another is using savestate to a write-protected file.   
+"""
+
+import seash
+import sys
+import os
+import seash_exceptions
+
+#pragma out Please load at least a pubkey, and work as that identity.
+#pragma out Don't use savestate to a write-protected file.
+
+open('write_protected_file', 'w+').close()
+os.system('chmod a-w write_protected_file')
+
+command_list = [ 
+  'savestate NO_KEYS_LOADED',
+]
+
+try:
+  seash.command_loop(command_list)
+except seash_exceptions.UserError, e:
+  print str(e)
+
+command_list = [
```

Comment:
Ideally, we would have the second test as a separate unit test file. Otherwise, you can't really say which of the tests failed.